src/background/chrome-api.ts

```diff
@@ -16,8 +16,9 @@ type Callback<Result> = (r: Result) => void;
  * This is exposed for testing. Consumers should use {@link chromeAPI}.
  */
 export function getChromeAPI(chrome = globalThis.chrome) {
-  // In the test environment, the `chrome` global may not exist.
-  if (typeof chrome === 'undefined') {
+  // In the test environment, the `chrome` global may not exist or may not
+  // be the expected object.
+  if (typeof chrome === 'undefined' || !chrome.extension) {
```

Comment:
This change fixes test failures I encountered on main when the `chromeAPI` variable in this file is initialized. Previously the `chrome` global did not exist at all in the ordinary web pages where the test runs. In Chrome Dev this global _is_ defined in that environment, but is missing most of the APIs that are expected in the extension environment.